docs/api/cozy-stack-client.md

```diff
@@ -101,6 +101,9 @@ query to work</p>
 <dt><a href="#isMatchingIndex">isMatchingIndex</a> ⇒ <code>boolean</code></dt>
```

Comment:
About the commit description : `it removes implicit operators`. It's more : `It adds explicit operators when they are implicit, typically the `$and` and `$eq` operators.